# HG changeset patch
# User Fabio Rossi  <rossi.f@inwind.it>
# Date 1339718908 14400
#      Thu Jun 14 20:08:28 2012 -0400
# Node ID 1c7956c661a8da0ea510cee5e56ffefec4516e43
# Parent  d213da1a16a6ed0823057eb92df607d187c9608c
Support spaces into the path to ghostscript.

* scripts/plot/private/__ghostscript__.m: Double quote output file.
(Bug # 36664).

diff --git a/scripts/plot/private/__ghostscript__.m b/scripts/plot/private/__ghostscript__.m
--- a/scripts/plot/private/__ghostscript__.m
+++ b/scripts/plot/private/__ghostscript__.m
@@ -130,17 +130,17 @@ function [gs_cmd, cleanup_cmd] = __ghost
     endif
   endif
 
   if (isempty (opts.binary))
     error ("print:no_ghostscript", "__ghostscript__.m: ghostscript is required.");
   elseif (isempty (opts.output))
     cmd = sprintf ("%s %s", opts.binary, gs_opts);
   else
-    cmd = sprintf ("%s %s -sOutputFile=%s", opts.binary, gs_opts, opts.output);
+    cmd = sprintf ("%s %s -sOutputFile=\"%s\"", opts.binary, gs_opts, opts.output);
   endif
   if (! isempty (opts.prepend)
       && any (strcmpi (opts.device, {"pswrite", "ps2write", "pdfwrite"})))
     ## FIXME - Fonts get may be mangled when appending ps/ps2.
     ##         See "How to concatenate several PS files" at the link,
     ##         http://en.wikibooks.org/wiki/PostScript_FAQ
     cmd = sprintf ("%s %s", cmd, opts.prepend);
     if (isempty (cleanup_cmd))
